src/palladio/NodeSpareParameter.cpp

```diff
@@ -15,49 +15,55 @@
  */
 
 #include "NodeSpareParameter.h"
+#include "SOPAssign.h"
 
-#include "PRM/PRM_SpareData.h"
 #include "OP/OP_Director.h"
 #include "PI/PI_EditScriptedParms.h"
-#include "SOPAssign.h"
+#include "PRM/PRM_SpareData.h"
+#include "UT/UT_VarEncode.h"
 
 #include "BoostRedirect.h"
 #include PLD_BOOST_INCLUDE(/functional/hash.hpp)
 
 namespace {
-const std::wstring parameterPrefix = L"_palladioAttribute_";
+constexpr wchar_t GROUP_SEPARATOR = L'.';

```

Comment:
this is duplicated from AttributeConversion.h (NameConversion NS)

Reply:
fixed